packages/core/src/components/Menu/MenuItem/components/MenuItemSubMenuIcon/MenuItemSubMenuIcon.module.scss

```diff
@@ -1,5 +1,6 @@
 .subMenuIconWrapper {
-  margin-left: var(--spacing-xs);
+  margin-left: auto;
+  padding-left: var(--spacing-xs);
```

Comment:
to be able to push the icon always to the right, regardless to the size of the content
as for today, MenuItem just adds flex-grow 1, which is ok, and then sub menu icon is pushed to the right, but we need something that **always** work, regardless of what the child is
we cannot expect people to implement flex-grow 1 for themselves, the sub menu icon should be pushed to the right by default